### PR TITLE
Major spell check and moderate wording updates

### PR DESCRIPTION
**Pull request by WolfgangSenff**

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -469,134 +469,134 @@ Every cleric can choose prayers from the basic prayers and from the prayer list
...
+When you see **d4**, **d6**, **d8**, **d10**, **d12** and **d20**, this is referring to a die of that size; d4 is a four-sided die and d20 is a twenty-sided die. Let's review all styles of dice notations we will use in this book:
 
 ### Adding dice (2d6) {.unlisted}
-And when you see **3d6** means to roll **three six-sided dice** and add the result.
+When you see **3d6**, that means roll **three six-sided dice** and add the result.
 
 ### Usage dice (Ud8) {.unlisted}
-Usage dice is a weird but very useful concept. Usage dice are used for tracking how many consumables a character has, like **Trowing Daggers(Ud8)** has a usage dice of a d8. Every time the consumable is used the usage dice should be rolled, if the result is a 1 or 2, that dice is downgraded to the smallest die in the following chain:
+Usage dice track consumable use. For instance, **Throwing Daggers (Ud8)** have a d8 usage die. Every time the consumable is used, the usage dice are rolled: if the result is a 1 or 2, that die is downgraded to the (next?) smallest value in the following chain:
```
Comment (WolfgangSenff) [inline source/index.md]: Wasn't sure if the chain should exist for a reason if it's not going to the next smallest, but always the smallest. I think next was intended here, will add if you give the go-ahead.

File: source/index.md [hunk 2]
```diff
@@ -469,134 +469,134 @@ Every cleric can choose prayers from the basic prayers and from the prayer list
...
 
-If the player is already going for the **extra effort** rolling 1 and 4 will not cause **fatigue**, since it's already receiving that point.
+If the player is already going for the **extra effort**, rolling a 1 or a 4 will not cause **fatigue**, since they are already receiving that point.
 
 Helping characters can also use the ***extra effort*** rule, rolling two dice instead of one, as long as they announce that before rolling.
 
 ### Zero Dice?
 
-If you for any reason need to roll zero dice, you roll 2 dice instead and keep the lowest number. Rolling less than zero dice is also possible, just keep adding dice, to roll -2 dice just roll 4 dice and choose the worst. I'm pretty sure that test will fail, though, sorry.
+If for any reason you need to roll zero dice, you roll 2 dice instead and keep the lowest number. Rolling less than zero dice is also possible: add dice to ensure you have 2 total dice being rolled. For instance, to roll -2 dice when you have only 1 dice available, roll 4 dice and choose the worst. I'm pretty sure that test will fail, though, sorry.
```
Comment (WolfgangSenff) [inline source/index.md]: I want to make sure this is accurate.

File: source/index.md [hunk 3]
```diff
@@ -839,24 +839,24 @@ When using a grid map, here's how you can convert distances":
...
 
 - **Decay tracker**: Represents the toll that dungeon delving takes on the **adventurers**. Causes hunger, exhaustion and other bad things to the players. Advances every turn.
-- **Doom tracker**: There's one for each dungeon, represents the dungeon "waking up" to the **adventurers**' presence and reacting.
-- **Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed something really bad happens.
+- **Doom tracker**: There's one for each dungeon, representing the dungeon "waking up" to the **adventurers**' presence and reacting.
+- **Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed, something really bad happens. (???? Are these two reversed?)
```
Comment (WolfgangSenff) [inline source/index.md]: I wasn't sure about this - it seems like these two values are reversed, but could be completely wrong.

File: source/dungeon/dungeon_level_2.md [hunk 4]
```diff
@@ -9,79 +9,77 @@ print: true
...
 - The other corpses inside the tubes twitch and look at the **adventurers** when they move around, but are harmless.
-- The green liquid is highly toxic, any contact with it will cause one **intelligence** wound (can't be reduced), and **nauseated** affliction.
+- The green liquid is highly toxic: any contact with it will cause one **intelligence** wound (can't be reduced), and the **nauseated** affliction.
 - 2 **ether** and 2 **fire** extract can be found here.
-- an **alchemic kit** is on the table.
-- A bottle of **wine of Yoros**, labeled on the floor.
-- There's a journal here, the last passage says: 
-  >"By absorbing corpses into the void, the weavers can certainly stop the dead vigil from happening. However, whenever they feed, everyone in the tomb feels sick and dizzy, even food rots and spoil. Worryingly, the area of effect is getting larger and larger with every corpse disposed. The question I have is whether there is a limit."
-
+- Nn **alchemic kit** is on the table.
```
Comment (saint11) [inline source/dungeon/dungeon_level_2.md]:
```suggestion
- An **alchemic kit** is on the table.
```

File: source/index.md (hunk 3 above)
Comment (saint11) [reply source/index.md]: I'm considering renaming it to "Bad Omens" tracker.
Honestly my logic is that the "Doom" just sounds cooler, and I wanted the cool thing to happen more often. But I might consider renaming both in the future.

```suggestion
- **Bad Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed, something really bad happens.
```

**Review by saint11 — APPROVED** [review]
This is great! Amazing work :)

File: source/index.md (hunk 1 above)
Comment (saint11) [reply source/index.md]: yes, next was intended!
```suggestion
Usage dice track consumable use. For instance, **Throwing Daggers (Ud8)** have a d8 usage die. Every time the consumable is used, the usage dice are rolled: if the result is a 1 or 2, that die is downgraded to the next smallest value in the following chain:
```

File: source/index.md (hunk 2 above)
Comment (saint11) [reply source/index.md]: I think this part is a bit confusing, I'll try to reword it for more clarity.

```suggestion
If, for any reason, you need to roll zero dice, you should roll 2 die instead and take the lowest number instead. Rolling less than zero dice is also possible: roll 2 die and add one extra dice for each value below zero. For instance, when rolling -3 die, roll 5 die (2+3) and choose the worst. I'm pretty sure that test will fail, though, sorry.
```

**Pull request merged**